charts/oauth2-proxy/templates/deployment.yaml

```diff
@@ -100,6 +100,9 @@ spec:
       {{- if .Values.terminationGracePeriodSeconds }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
       {{- end }}
+{{- if .Values.extraInitContainers }}
```

Comment:
Maybe a nice to have is to add extra comment here so we know that this is our custom implementation and needs to be re-added if we update the chart